### PR TITLE
Convert to ecmarkup

### PR DESCRIPTION
**Pull request by sophiebits**
See it at http://spicyj.github.io/ecmascript-rest-spread/. You should probably make sure I wrote all the right things.

### CODE REVIEW

File: Spec.html [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+  <h2>Syntax</h2>
+
+  <emu-production name="PropertyDefinition" params="Yield">
+    <emu-rhs><emu-nt params="?Yield">IdentifierReference</emu-nt></emu-rhs>
+    <emu-rhs><emu-nt params="?Yield">CoverInitializedName</emu-nt></emu-rhs>
+    <emu-rhs><emu-nt params="?Yield">PropertyName</emu-nt><emu-t>:</emu-t><emu-nt params="In, ?Yield">AssignmentExpression</emu-nt></emu-rhs>
+    <emu-rhs><emu-nt params="?Yield">MethodDefinition</emu-nt></emu-rhs>
+    <ins class="block">
+      <emu-rhs><emu-t>...</emu-t><emu-nt params="In, ?Yield">AssignmentExpression</emu-nt></emu-rhs>
```
Comment (sophiebits) [inline Spec.html:42]: I added a [In, ?Yield] here to match the "PropertyName: AssignmentExpression" case above. Not sure if this is correct.

File: Spec.html [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+  <h1>Rest Properties</h1>
+
+  <h2>Syntax</h2>
+
+  <emu-production name="ObjectAssignmentPattern" params="Yield">
+    <emu-rhs><emu-t>{</emu-t><emu-t>}</emu-t></emu-rhs>
+    <emu-rhs><emu-t>{</emu-t><emu-nt params="?Yield">AssignmentPropertyList</emu-nt><emu-t>}</emu-t></emu-rhs>
+    <emu-rhs><emu-t>{</emu-t><emu-nt params="?Yield">AssignmentPropertyList</emu-nt><emu-t>,</emu-t><emu-t>}</emu-t></emu-rhs>
+    <ins class="block">
+      <emu-rhs><emu-t>{</emu-t><emu-t>...</emu-t><emu-nt params="?Yield">DestructuringAssignmentTarget</emu-nt><emu-t>}</emu-t></emu-rhs>
```
Comment (sophiebits) [inline Spec.html:74]: Likewise, a bunch of [?Yield] subscripts added here.

File: Spec.html [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    <emu-rhs><emu-nt params="?Yield">MethodDefinition</emu-nt></emu-rhs>
+    <ins class="block">
+      <emu-rhs><emu-t>...</emu-t><emu-nt params="In, ?Yield">AssignmentExpression</emu-nt></emu-rhs>
+    </ins>
+  </emu-production>
+
+  <emu-clause id="Spread-RuntimeSemantics-PropertyDefinitionEvaluation">
+    <h1>Runtime Semantics: PropertyDefinitionEvaluation</h1>
+
+    <p>With parameters _object_ and _enumerable_.</p>
```
Comment (sophiebits) [inline Spec.html:49]: "enumerable" wasn't in your markdown, but the spec I see has it. Does this need a `Assert: _enumerable_ is *true*.` like some other cases have?

File: Spec.html [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+      1. <del>Return</del> <ins>Let _nextNames_ be</ins> the result of performing DestructuringAssignmentEvaluation for |AssignmentProperty| using _value_ as the argument.
+      1. <ins>ReturnIfAbrupt(_nextNames_).</ins>
+      1. <ins>Append each item in _nextNames_ to the end of _propertyNames_.</ins>
+      1. <ins>Return _propertyNames_.</ins>
+    </emu-alg>
+
+    <emu-grammar>AssignmentProperty : IdentifierReference Initializer?</emu-grammar>
+    <emu-alg>
+      1. Let _P_ be StringValue of |IdentifierReference|.
+      1. Let _lref_ be ? ResolveBinding(_P_).
```
Comment (sophiebits) [inline Spec.html:144]: You had ReturnIfAbrupt but I converted to ? to match the current spec here and in CopyDataProperties.

File: Spec.html (hunk 3 above)
Comment (sebmarkbage) [inline Spec.html:49]: Yea, I think this is probably new.

File: Spec.html (hunk 1 above)
Comment (sebmarkbage) [inline Spec.html:42]: Yea I think that's correct.

**Comment by sebmarkbage** [comment]
Thanks @spicyj!

**Pull request merged**